v1/validator.go

```diff
@@ -29,14 +29,24 @@ func GetSubMux() *goji.Mux {
 	v1.HandleFunc(pat.Get("/"), info)
 	v1.HandleFunc(pat.Post("/validate"), validate)
 
+	v1.HandleFunc(pat.Get("/validate"), func(writer http.ResponseWriter, _ *http.Request) {
+		writer.WriteHeader(405)
+	})
+	v1.HandleFunc(pat.Get("/validate/"), forwardToValidate)
+	v1.HandleFunc(pat.Post("/validate/"), forwardToValidate)
+
 	return v1
 }
 
+func forwardToValidate(writer http.ResponseWriter, request *http.Request) {
+	http.Redirect(writer, request, "/v1/validate", 302)
+}
+
 func info(writer http.ResponseWriter, request *http.Request) {
 	serverInfo := serverInfo{
 		Description: "Space API Validator API",
-		Usage:       "Send a POST request in JSON format to /v1/validate/. See https://github.com/SpaceApi/validator for more information.",
-		Version:     "1.0.0",
+		Usage:       "Send a POST request in JSON format to /v1/validate. See https://github.com/SpaceApi/validator for more information.",
+		Version:     "2.0.0",
```

Comment:
Hm, I was thinking of something like `1.1.0`. It's a bit weird to say that this is version 2.0.0 of the v1 API.

Reply:
done